README.md

```diff
@@ -32,7 +32,7 @@ Tool that generates custom [Angular-Material v1.x](http://github.com/angular/mat
 | ----------------------- | ----------- | -------------------------------------------------------------------------- |
 | `destination` (*)       | `string`    | Target location for the Material build.                                    |
 | `modules`               | `string[]`  | Modules that should be part of the build.<br/> All modules will be built if nothing is specified.                                                                                                           |
-| `version`               | `string`    | Version of Angular Material.<br/> If set to local, it will take the local installed Angular Material version from the node modules                                                                               |
+| `version`               | `string`    | Version of Angular Material.<br/> If set to `local`, it will take the local installed Angular Material version from the node modules. If set to `latest`, the latest version will be downloaded.                                                           |
```

Comment:
Probably add a line-break here. As same as we do sometimes in the ngMaterial docs